src/components/Histogram/HistogramComponent.tsx

```diff
@@ -151,7 +151,7 @@ export class HistogramComponent extends React.Component<WidgetProps> {
         // Update widget title when region or coordinate changes
         autorun(() => {
             const appStore = this.props.appStore;
-            if (this.widgetStore && appStore.selectedRegion) {
+            if (appStore.selectedRegion || !appStore.selectedRegion) {
```

Comment:
this line makes no sense to me...

Reply:
can't the region ID be left as "Active" (perhaps we can use -3 or something for that?) and then the rest of the values can be computed from that, rather than a whole bunch of autoruns?

Reply:
> this line makes no sense to me...

Sorry about that. It was just a temporary workaround for getting image/cursor data. Since `appStore.selectedRegion` always pass `null` when deselect a region so the regionId of a "Active" widget won't change to -1(image) or 0(cursor).

Reply:
> can't the region ID be left as "Active" (perhaps we can use -3 or something for that?) and then the rest of the values can be computed from that, rather than a whole bunch of autoruns?

Yes, it can. I just haven't thought that before...:joy: 
I have made a new commit for what you suggested. It's truly a better way.
Thanks